client/signup/steps/jpo-contact-form/style.scss

```diff
@@ -0,0 +1,121 @@
+.jpo__contact-form-wrapper {
+	margin-left: auto;
+	margin-right: auto;
+	display: flex;
+	max-width: 325px;
+}
+
+.jpo__contact-form-wrapper .card {
+	max-width: 320px;
+	text-align: center;
+}
+
+.jpo-contact-form__description {
+	font-size: 1.25rem;
+	color: $gray;
+}
+
+.jpo-contact-form__image {
+	padding:15px;
+}
+
+.jpo-contact-form__choice-copy {
+	padding: 15px;
+	border-top: 1px solid transparentize( lighten( $gray, 20% ), .5 );
```

Comment:
We should make sure the borders and shadows in this component match `jpo-site-type/style.scss` after my suggestions have been implemented. Even better if we make most of this stuff into reusable components.